src/pages/blog/2022-04-25-ako-na-monero-node-na-raspberry-pi.md

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Ako na Monero node na Raspberry Pi'
+author: Monika Jassova
+cover: /assets/monero-node-header.png
+tags:
+  - návod
+---
+
+*Príspevok zhŕňa zdroje, podľa ktorých som postupovala pri inštalácii Monero nodu (uzla) na Raspberry Pi, používanie s Torom a poznatky z tohto procesu.*
+
+## Prečo bežať vlastný node?
+- vyššia bezpečnosť: celý blockchain sa sťahuje na tvoje zariadenie a všetky transakcie a blocky sa lokálne overujú. Pripojená peňaženka pre účely transakčnej histórie skenuje kópiu blockchainu na lokálnom node namiesto spoliehania sa na tretiu stranu.
+- prispievaš k decentralizácii siete
+
+### ...a prečo ho používať v kombinácii s Tor/VPN?
+- vyššie súkromie: pri vysielaní do Monero siete je IP adresa odhalená, preto node, ktorý obdrží transakciu, môže identifikovať fyzickú lokáciu/identitu odosielateľa. Traffic cez nody je viditeľný aj pre poskytovateľov internetu a ďalšie strany vykonávajúce sledovanie, preto analýza trafficu nodu(-ov), ktoré obdržia viaceré transakcie pochádzajúce z rovnakej IP adresy, môže viesť k odhaleniu, že sú transakcie prepojené.
+- vyhnutie sa potenciálnej cenzúre transakcií
+
+**[Tor](https://sk.wikipedia.org/wiki/Tor_(sie%C5%A5))**: decentralizovaná open-source privátna sieť vyvinutá tak, aby žiaden účastník nevedel identifikovať zdroj konkrétneho vysielania.</br>
+**[VPN](https://sk.wikipedia.org/wiki/Virtu%C3%A1lna_priv%C3%A1tna_sie%C5%A5)** (virtuálna privátna sieť): posiela traffic cez šifrovaný tunel, ktorý poskytovatelia internetu a vlády nevedia sledovať. Dôvera sa presúva na poskytovateľov VPN, ktorí však môžu držať logy.
+
+## Čo potrebujeme
+- https://github.com/monero-ecosystem/PiNode-XMR Monero node pre jednodoskové počítače s webovým rozhraním a predkonfigurovanými doplnkovými nástrojmi a s jednoduchou inštaláciou
+- jednodoskový počítač: je lacný a má nízku spotrebu energie. Prehľad podporovaných HW [tu](https://github.com/monero-ecosystem/PiNode-XMR/wiki/Hardware#hardware), najrozšírenejšie a asi najľahšie zohnateľné je Raspberry Pi (ja som použila model 4B s 4GB RAM), autor PiNode odporúča Odroid s podporou AES pre rýchlejšie de/šifrovanie a Rock64 pre najlepší pomer cena/výkon.
```

Comment:
Lepsie je dat odkaz na cele slovo/frazu ako len na slovo "tu". Je to viac pouzitelne pre citatela.

```suggestion
- jednodoskový počítač: je lacný a má nízku spotrebu energie. [Prehľad podporovaných HW](https://github.com/monero-ecosystem/PiNode-XMR/wiki/Hardware#hardware), najrozšírenejšie a asi najľahšie zohnateľné je Raspberry Pi (ja som použila model 4B s 4GB RAM), autor PiNode odporúča Odroid s podporou AES pre rýchlejšie de/šifrovanie a Rock64 pre najlepší pomer cena/výkon.
```

Reply:
opravene